.github/workflows/test.yaml

```diff
@@ -200,6 +200,11 @@ jobs:
       run: |
         cd ${{ github.workspace }}/src/github.com/snapcore/snapd || exit 1
         ./run-checks --unit
+    - name: Test Go (SNAPD_DEBUG=1)
+      if: steps.cached-results.outputs.already-ran != 'true'
+      run: |
+        cd ${{ github.workspace }}/src/github.com/snapcore/snapd || exit 1
+        SNAPD_DEBUG=1 ./run-checks --unit
```

Comment:
Actually one small question/nitpick - all the other unit tests have `SKIP_DIRTY_CHECK=1` - is there a reason why we do not add it here as well?